src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +33,26 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
```

Comment:
We may want to change the `Global.OKAPI_TO_URL` to `Global.OKAPI_URL_TO`. The way it is now is a bit confusing.

Reply:
I have changed it